exporters/Containerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.access.redhat.com/ubi8/python-39
+
+ARG WORK_DIR=/app/pelorus
+ARG PELORUS_IMAGE_TAG=""
+ARG OPENSHIFT_BUILD_SOURCE=""
+ARG OPENSHIFT_BUILD_REFERENCE=""
+
+ENV APP_FILE=deploytime/app.py \
+    PELORUS_IMAGE_TAG=$PELORUS_IMAGE_TAG \
+    OPENSHIFT_BUILD_SOURCE=$OPENSHIFT_BUILD_SOURCE \
+    OPENSHIFT_BUILD_REFERENCE=$OPENSHIFT_BUILD_REFERENCE
+
+USER root
+
+COPY ./ $WORK_DIR
+
+RUN dnf -y install skopeo && \
```

Comment:
Do we need to clean cache in this layer? In debian images I usually run `rm -rf /var/lib/apt/lists/*` after `apt-get install`

Reply:
yes, any cleanup done in this step will reduce it's size. But it needs to be within line 17.